src/it/cert-manager/src/test/java/io/kokuwa/maven/k3s/PodIT.java

```diff
@@ -0,0 +1,11 @@
+package io.kokuwa.maven.k3s;
+
+import org.junit.jupiter.api.Test;
+
+public class PodIT {
+
+	@Test
+	void test() {
+		// the test should only trigger the deployment and assert it does not fail because of the completed job.
```

Comment:
This can be removed?

Reply:
I thought an empty test is required to actually trigger execution(like in the statefulset it). Comment is only for clarification

Reply:
If you do not have failsafe configured this class will be ignored anyway.

Reply:
Isnt failsafe inherited from the parent? But I can remove it, no problem.

Reply:
meh, please ignore my comment